pkg/extensions/function/cmap.go

```diff
@@ -50,6 +51,10 @@ const appInstanceLabel = "app.kubernetes.io/instance"
 var configMapGVK = corev1.SchemeGroupVersion.WithKind("ConfigMap")
 
 func (r *Reconciler) reconcileConfigmap(ctx context.Context) error {
+	if skip.Skip(ctx) {
+		return nil
+	}
```

Comment:
I put this here because `pkg/reconciler/testing.TestReconcileAdapter()` only performs assertions on events related to the reconciliation of the adapter.

This reconciler additionally manages a ConfigMap, so tests would fail with unexpected extra actions ("configmap created") if we didn't skip that part.

In production, this is a no-op.
We use this in a few other components (mostly sources).